lib/xcodeproj/scheme/launch_action.rb

```diff
@@ -62,6 +62,28 @@ def buildable_product_runnable=(runnable)
         @xml_element.add_element(runnable.xml_element) if runnable
       end
 
+      # @return [EnvironmentVariables]
+      #        Returns the set of environment variables that are currently set at test runtime
+      #        NOTE: If this creates a new node, it will not automatically be added to the xml document
```

Comment:
I removed this, as it seems to be consistent with the other method behaviors, so appears to be expected.